packages/fx-core/src/common/tools.ts

```diff
@@ -451,3 +451,37 @@ export function getAppStudioEndpoint(): string {
     return "https://dev.teams.microsoft.com";
   }
 }
+
+export async function copyFiles(
+  srcPath: string,
+  distPath: string,
+  excludeFileList?: { fileName: string; recursive: boolean }[]
```

Comment:
Nit: give default value to exclude `undefined` case
```
excludeFileList: { fileName: string; recursive: boolean }[] = []
```

Reply:
Good idea, thx.